src/nanosvg.h

```diff
@@ -29,6 +29,33 @@
 #ifndef NANOSVG_H
 #define NANOSVG_H
 
+#include <math.h>
```

Comment:
This should go into the implementation section (see where math.h is already included). Please only define the functions that are actually used.